axum-core/src/macros.rs

```diff
@@ -7,7 +8,6 @@ macro_rules! __log_rejection {
         body_text = $body_text:expr,
         status = $status:expr,
     ) => {
-        #[cfg(feature = "tracing")]
         {
             tracing::event!(
                 target: "axum::rejection",
```

Comment:
Hmm, I get where you're coming from re. confusion but actually I find it also pretty weird that right now, we use this tracing target for both `axum` and `axum-extra` extractors... I'm wondering if it wouldn't be better to just copy-paste the macro into each crate.

Reply:
Okay, let's merge this then.